src/components/attendees/AttendeeTable.tsx

```diff
@@ -35,68 +51,170 @@ export default function AttendeeTable() {
+										<div className="px-8 py-4 text-xl font-bold flex items-start justify-start gap-4">
+											<button
+												className="h-8 w-8 bg-gray-200 rounded-full flex items-center justify-center text-attendee-name"
+												onClick={() => {
+													setPopUp((popUp) => ({
+														...popUp,
+														open: false,
+													}));
+												}}
+											>
+												<CloseIcon />
+											</button>
+											<div className=" text-popup-heading text-bold text-xl">
+												{`${popUp.selectedAttendee.firstName} ${popUp.selectedAttendee.lastName}`}
+											</div>
+										</div>
+										<AttendeePopUp
+											selectedAttendee={
+												popUp.selectedAttendee
+											}
+										/>
+									</div>
+								</Dialog.Panel>
+							</Transition.Child>
+						</div>
+					</div>
+				</Dialog>
+			</Transition.Root>
+			{/* Search Component */}
+			<div className="h-auto w-full px-5">
+				<h1 className=" font-bold text-3xl text-subHeading-color mb-4">
+					Attendees
+				</h1>
+				<input
+					id="search"
+					className="mb-8"
+					type={'text'}
+					placeholder="Search Attendees"
+					onChange={(e) => {
+						setSearch(e.target.value);
+					}}
+				/>
+			</div>
+			{/* Table Component */}
+			<div className="h-full w-full overflow-auto  p-5">
+				<table className="w-full border-collapse border-spacing-0 p-5">
+					<tbody>
+						<tr>
+							<th>
+								<div id="nameContent">
+									Name
+									{!namesAscending ? (
+										<button>
+											<SortIcon id="ascendingSort" />
+										</button>
+									) : (
+										<button>
+											<SortIcon id="descendingSort" />
+										</button>
+									)}
+								</div>
+							</th>
+							<th> Email </th>
+							<th> Discord </th>
+							<th> Auth </th>
+							<th> School </th>
+							<th> Accepted </th>
+							<th> Confirmed </th>
+							<th> Checked In </th>
+							<th id="filterIcon" className="">
+								<div className=" flex w-full justify-end align-center">
+									<FilterIcon />
+								</div>
+							</th>
+						</tr>
+						{attendeeData.map((attendee) => {
+							return (
+								<tr key={attendee.id}>
+									<AttendeeRow
+										key={attendee.id}
+										attendee={attendee}
+									/>
+									<td>
+										<button
+											id="openUser"
+											onClick={() => {
+												if (!popUp.seenAttendeePopUp) {
+													setPopUp((popUp) => ({
+														...popUp,
+														seenAttendeePopUp: true,
+														selectedAttendee:
+															attendee,
+														open: true,
+													}));
+												} else {
+													setPopUp((popUp) => ({
+														...popUp,
+														selectedAttendee:
+															attendee,
+														open: true,
+													}));
+												}
+											}}
+										>
+											<div id="openUserContent">
+												<OpenIcon />
+												<div> Open User </div>
```

Comment:
The div isn't doing anything here as far as I can tell. This can just be the text, or a `<p>` if there is eventually going to be some styling here.

Reply:
I can remove it!